contracts/test/integration/LoanRepayment.t.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Vm} from "./../unit/utils/Vm.sol";
+import {DSTest} from "./../unit/utils/DSTest.sol";
+import {getCore, getAddresses} from "./../unit/utils/Fixtures.sol";
+import {IVolt, Volt} from "../../volt/Volt.sol";
+import {OtcEscrow} from "./../../utils/OtcEscrow.sol";
+import {MainnetAddresses} from "./fixtures/MainnetAddresses.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+contract IntegrationTestLoanRepayment is DSTest {
+    IVolt private volt = IVolt(MainnetAddresses.VOLT);
+    IVolt private fei = IVolt(MainnetAddresses.FEI);
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+
+    /// @notice amount of fei to pay off loan
+    uint256 public constant feiAmount = 10_170_000e18;
+
+    /// @notice amount of volt to receive upon repayment
+    uint256 public constant voltAmount = 10_000_000e18;
+
+    /// @notice escrow contract
+    OtcEscrow public otcEscrow;
+
+    address public feiDaoTimelock = MainnetAddresses.FEI_DAO_TIMELOCK;
+    address public voltTimelock = MainnetAddresses.VOLT_TIMELOCK;
+
+    function setUp() public {
+        otcEscrow = new OtcEscrow(
+            MainnetAddresses.FEI_DAO_TIMELOCK, /// beneficiary
+            MainnetAddresses.VOLT_TIMELOCK, /// recipient
+            MainnetAddresses.VOLT, /// received token
+            MainnetAddresses.FEI, /// sent token
+            voltAmount, /// received amount
+            feiAmount /// sent amount
+        );
+
+        if (volt.allowance(feiDaoTimelock, address(otcEscrow)) < voltAmount) {
+            vm.prank(feiDaoTimelock);
+            volt.approve(address(otcEscrow), voltAmount);
+        }
+        if (fei.balanceOf(address(otcEscrow)) < feiAmount) {
+            vm.prank(MainnetAddresses.GOVERNOR);
+            fei.transfer(address(otcEscrow), feiAmount);
+        }
```

Comment:
Tiny nit, but we could check the difference in balance here, & only send the required amount to complete the amount in the escrow contract, as opposed to sending the entire amount.

Reply:
Otc Escrow will have 0 fei to start, and the only tx to fund it will include the full amount.